cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -282,23 +282,20 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
     val tx = db.beginTransaction(Type.explicit, AnonymousContext.read())
     try {
       val nodes = tx.findNodes(Label.label(label)).asScala.toList
-      // TODO: Remove this conditional once we have system graph initialization working OK
-      if (nodes.nonEmpty) {
-        nodes.length should be > 0
-        val props = nodes.map(n => n.getProperty("name"))
-        val result = p.columnAs[String](column).toList
-        result should equal(props)
-      }
+      nodes.length should be > 0
+      val props = nodes.map(n => n.getProperty("name"))
+      val result = p.columnAs[String](column).toList
+      result.toSet should equal(props.toSet)
     } finally {
       tx.close()
     }
   })
 
-  private def assertRolesShown(expected: Seq[String] = List.empty) = ResultAndDbAssertions((p, db) => {
-    val tx = db.beginTransaction(Type.explicit, AnonymousContext.read())
```

Comment:
@craigtaverner I had to fix a merge conflict here (see the 2nd commit) - just wanted to check with you that this isn't going to cause problems

Reply:
My commit did not change the line for beginTransaction, so I assume whatever was the version for 4.0 should be the right one. For 4.1 I re-ran all tests before pushing. If all tests pass here, we should be fine. This test method is only used in one test, so it might be worth reviewing the output of that as well, in case it has changed. One suspicion I have is the addition of the PUBLIC role, which is a 4.1 only change, will make the output table look cluttered. This is completely independent of this commit, but worth considering when reviewing the 4.1 docs.

Reply:
See my final comment below. The beginTransaction line is wrong.